app/uk/gov/hmrc/incometaxsessiondata/services/SessionService.scala

```diff
@@ -38,4 +38,8 @@ class SessionService @Inject()(
   def set(sessionData: SessionData): Future[Boolean] = {
     repository.set(sessionData)
   }
+
+  def deleteSession(sessionId: String): Future[Boolean] = {
```

Comment:
if we can apply same approach as in this discussion:

https://hmrcdigital.slack.com/archives/C050V2P86BC/p1712737921076559?thread_ts=1712737564.343879&cid=C050V2P86BC

Reply:
Hi Pavel, do you mean just change the return type to Future[Unit]? I use the same approach as Siddharth on the repository level already

Reply:
I have done this now